lua/textcase/plugin/plugin.lua

```diff
@@ -23,6 +24,8 @@ function M.register_methods(method_table, opts)
   -- TODO: validate method_table
   M.state.methods_by_desc[method_table.desc] = method_table
```

Comment:
The `methods_by_desc` is not used anymore after this change. So it could be removed.